src/test/java/com/splunk/hecclient/IndexerTest.java

```diff
@@ -194,4 +194,26 @@ private Indexer assertFailure(CloseableHttpClient client) {
         Assert.assertEquals(indexer.getChannel(), poller.getChannel());
         return indexer;
     }
+
+    @Test
+    public void sendCompressedBatchWithSuccess() {
```

Comment:
Can we add another test for sending RAW successfully?

Reply:
Added Test case for Raw batch

Reply:
It's already covered in this test case, so I don’t think to add new test case